sedai/README.md

```diff
@@ -1,59 +1,42 @@
-* Achieve 99.99% or better application availability
-* Autonomously reduce resource latency by 75%
-* Reduce cloud costs by 30% through autonomous optimization
-* Increase team productivity and agility by 3x
+### How it Works
 
-### Core features
+* **Agentless:** Seamlessly connects to your cloud accounts and automatically discovers and understands production environments.
 
-* **Easy setup:** Sedai's agentless approach integrates with your cloud accounts and observability platforms so you can get up and running in minutes.
-* **Configuration-free:** By inferring your topology and metrics, Sedai leverages them to detect signals based on observed performance.
-* **Autonomous Availability:** Prevent issues and ensure the highest levels of availability with autonomous actions.
-* **Release Intelligence:** Validate release performance in production in real-time with smart scorecards that assess the quality of your deployments and analyze deviations in latency and errors.
-* **Resource Optimization:** Improve customer experience and lower cloud costs on the fly with dynamic allocation that manages your resources based on traffic needs.
-* **Smart SLOs:** In addition to automatically analyzing and defining targets, Sedai manages error budgets with autonomous actions that prevent errors, downtime, and SLO breaches.
+* **Configuration-free:** Easily connects to Datadog API and other APM/Observability platforms, and intelligently identifies, prioritizes and learns metric behavior.
 
-To learn more, see [Sedai Inc.][1]
+* **Proactive Actions:** Safely acts in production on your behalf to ensure that resources avoid availability issues and run optimally at all times.
 
 ## Setup
 
-After connecting your cloud provider, monitoring provider, and notification provider accounts, Sedai identifies your topology and metrics and starts to analyze performance.
-
-### Installation
-
-Existing Sedai customers can receive notifications about Sedai activity in Datadog. For more information, see the [Sedai Notifications][6].
+In Sedai:
 
-## Data Collected
-
-### Metrics
+* Navigate to Settings > Notifications > Add Integration > DataDog Icon 
+![DataDog Notification Integration](images/DataDog_Notification_Integration.png)
 
-This integration does not collect metrics.
+* Enter a Nick Name and the API Key for your DataDog account, Enable it and Test the integration.
+![DataDog Notification Integration](images/Add_DataDog_Channel.png)
 
-### Events
+* Once the Test is verified to be working, click Save.
+![Working DataDog Integration](images/Add_DataDog_Channel-Working_REC.png)
 
-This integration does not support events.
+* Under Settings > Notifications, [select which notifications][2] you want to send to Datadog. 
+![Enable DataDog Notifications](images/Enable_Notifications.png)
 
-### Service
+## Data Collected
 
-This integration does not collect service checks.
+This integration does not collect metrics or service checks, and does not support events.
```

Comment:
Apologies if we've already discussed this, but isn't Sedai sending data into Datadog through events? If not, what kind of data is being sent in?

Reply:
Hi @anawishnoff,
Yes, you are right - we send in Notifications on Sedai events that the user has enabled in set up steps to Datadog through the Events API.
Do you think the following line would do..? If not, could you please share your suggestion/recommendation..?

"This integration sends notifications on subscribed events from Sedai to Datadog."